Documentation/helm-operator.md

```diff
@@ -88,40 +88,57 @@ The command removes all the Kubernetes components associated with the chart and
+| `agent.flexVolumeDirPath`       | Path where the Rook agent discovers the flex volume plugins (*)                                         | `/usr/libexec/kubernetes/kubelet-plugins/volume/exec/` |
+| `agent.libModulesDirPath`       | Path where the Rook agent should look for kernel modules (*)                                            | `/lib/modules`                                         |
+| `agent.mounts`                  | Additional paths to be mounted in the agent container (**)                                              | <none>                                                 |
+| `agent.mountSecurityMode`       | Mount Security Mode for the agent.                                                                      | `Any`                                                  |
+| `agent.toleration`              | Toleration for the agent pods                                                                           | <none>                                                 |
+| `agent.tolerationKey`           | The specific key of the taint to tolerate                                                               | <none>                                                 |
+| `agent.tolerations`             | Array of tolerations in YAML format which will be added to agent deployment                             | <none>                                                 |
+| `agent.nodeAffinity`            | The node labels for affinity of `rook-agent` (***)                                                      | <none>                                                 |
+| `csi.enableRbdDriver`           | Enable Ceph CSI RBD driver.                                                                             | `true`                                                 |
+| `csi.enableCephfsDriver`        | Enable Ceph CSI CephFS driver.                                                                          | `true`                                                 |
+| `csi.enableGrpcMetrics`         | Enable Ceph CSI GRPC Metrics.                                                                           | `true`                                                 |
+| `csi.provisionerTolerations`    | Array of tolerations in YAML format which will be added to CSI provisioner deployment.                  | <none>                                                 |
+| `csi.provisionerNodeAffinity`   | The node labels for affinity of the CSI provisioner deployment (***)                                    | <none>                                                 |
+| `csi.pluginTolerations`         | Array of tolerations in YAML format which will be added to Ceph CSI plugin DaemonSet                    | <none>                                                 |
+| `csi.pluginNodeAffinity`        | The node labels for affinity of the Ceph CSI CephFS and RBD plugin DaemonSet (***)                      | <none>                                                 |
```

Comment:
`Ceph CSI CephFS and RBD plugin` to `Ceph CSI plugin` as above in `csi.pluginTolerations`

Reply:
Fixed.